credentials/apps/api/v2/serializers.py

```diff
@@ -322,7 +357,46 @@ def create(self, validated_data):
             defaults={
                 "is_active": validated_data["is_active"],
                 "course_run": course_run,
-                "certificate_available_date": validated_data["certificate_available_date"],
+                "certificate_available_date": validated_data.get("certificate_available_date"),
+                "title": validated_data.get("title"),
             },
         )
+
+        cert.signatories.all().delete()
```

Comment:
Here's what I thought here. Looks Signatory has a many to many relation. If so, can we delete everything here like this? What do you think?

Reply:
I mean that this is a many-to-many relation already in the design of the database table and the design is not ours. This means that several certificates can look into the same signature. What are we doing? we delete for one and disappear for all.

Probably we have to consider get_or_create if it's possible. Please draw attention to this.

Reply:
Unfortunately, we cannot use get_or_create here because the Signatory model contains an ImageField that is extremely problematic to get for in the database, and the rest of the fields are not unique.
Therefore, for now, I suggest only deleting the relation between existing signatures.

Reply:
>the rest of the fields are not unique

Why do we need them to be unique? We have `name`, `title`. You can use them. And then we have m2m, and we behave like o2m.